lib/roger/system.ex

```diff
@@ -122,7 +160,7 @@ defmodule Roger.System do
 
   def handle_info(:check_started_partitions, state) do
     Process.send_after(self(), :check_started_partitions, 1000)
-    if state.channel != nil do
+    if state.channel != nil && state.active do
```

Comment:
`!= nil` is maybe redundant since `state.channel` would be falsy/truthy in this check